crates/rattler_solve/src/libsolv/wrapper/solver.rs

```diff
@@ -87,6 +150,10 @@ impl Solver<'_> {
             // Safe because we know the `transaction` ptr is valid
             Ok(unsafe { Transaction::new(self, transaction) })
         } else {
+            let problems = self.all_solver_problems();
+            println!("problems: {:#?}", problems);
```

Comment:
Left over `println`

Reply:
yeah I used it for testing. Shoudl we keep the PR WIP and leave it in, or try to merge it already?

Reply:
Wip is fine 👍

Reply:
I removed it for now